supplementary_style_guide/glossary_terms_conventions/general_conventions/p.adoc

```diff
@@ -31,6 +31,16 @@
 
 *See also*:
 
+[discrete]
+[[persistent-storage]]
+==== image:images/yes.png[yes] persistent storage (noun)
+*Description*: A type of storage that persists data even after the computer is turned off or after the process that created the data ended, for example hard disk, tape, or flash memory.
```

Comment:
I think persist is an intransitive verb, in which case the storage can't _persist_ the data. It could preserve it or retain it though because those are transitive. Also in another PR we changed the glossary entries to complete sentences, which would make sense throughout this one too. So here maybe something like this would work:

`Persistent storage is type of storage that preserves data even after the computer is turned off or after the process that created the data ends. A hard disk, tape, or flash memory are examples of persistent storage.`

This suggestion also makes the last clause a separate sentence because it read as if those were examples of data ending, not persistent storage itself.